gns3/modules/virtualbox/dialogs/virtualbox_vm_wizard.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, virtualbox_vms, parent):
             # we want to see the cancel button on OSX
             self.setOptions(QtWidgets.QWizard.NoDefaultButton)
 
-        if VirtualBox.instance().settings()["use_local_server"]:
+        if Servers.instance().isNonLocalServerConfigured() is False:
```

Comment:
VirtualBox can only be used on a local server or remote. If the GNS3 VM is enabled and not remote server configured, we get the first page where we can only select the local server option.